cmd/ethrex/utils.rs

```diff
@@ -111,8 +113,20 @@ pub fn parse_socket_addr(addr: &str, port: &str) -> io::Result<SocketAddr> {
         ))
 }
 
-pub fn set_datadir(datadir: &str) -> String {
-    let project_dir = ProjectDirs::from("", "", datadir).expect("Couldn't find home directory");
+pub fn set_datadir(datadir: &str, network: &Option<String>) -> String {
+    let sub_path = match network {
+        Some(network) => {
+            if DEFAULT_PUBLIC_NETWORKS.contains(&network.as_str()) {
+                &network.clone()
+            } else {
+                &(String::from(DEFAULT_CUSTOM_DIR).to_owned() + &network.clone())
+            }
+        }
+        None => &(String::default()),
```

Comment:
either network is a mandatory param, or we should default to mainnet. But we shouldn't get to this line.